cmd/tailscale/cli/switch.go

```diff
@@ -34,6 +34,22 @@ This command is currently in alpha and may change in the future.`,
 		return fs
 	}(),
 	Exec: switchProfile,
+
+	// Add remove subcommand
+	Subcommands: []*ffcli.Command{
```

Comment:
Hmm, I'd be tempted to make it a flag for the switch command (`tailscale switch -remove <id>`) since that feels more consistent with `-list`, but it sounds like previous conversations suggested it should be a subcommand so I guess that works well enough too. A subcommand is probably more consistent with how other parts of the CLI operate.

@bradfitz is on vacation so I'll just say the subcommand in its current form is fine.